thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/libs/handlebarsHelpers.js

```diff
@@ -29,6 +29,17 @@ Handlebars.registerHelper('formatDelta', function (a, b) {
     return "" + delta.toFixed(1);
   }
 });
+
+/**
+ * Template helper that assigns the correct css class based on the delta
+ * @param  {string} value: string value of delta ('2.4%')
+ * @return [string] css class positive/negative
+ */
+Handlebars.registerHelper('colorDelta', function(value) {
+    if (value ===  'N/A') return;
```

Comment:
I was actually looking through our style guide and airbnb's and couldn't really find a specific guildeline for single line-if. Closest I found is this: https://github.com/airbnb/javascript#blocks--braces